.github/workflows/release.yml

```
@@ -4,79 +4,109 @@ on:
-        with:
-          python-version: "3.11"
+          ref: ${{ github.sha }}
 
-      - name: Validate component metadata
+      - name: Validate release metadata
         id: package
         shell: bash
-        env:
-          EVENT_NAME: ${{ github.event_name }}
-          DISPATCH_TAG: ${{ inputs.tag }}
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2129:style:11:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)